ch3/01.py

```diff
@@ -0,0 +1,8 @@
+import random
+
+lista = []
+
+for x in range(3):
```

Comment:
range() vs xrange(), in python < 3.0 there are both, in that case you use xrange(), you know why?